apps/aecore/test/aecore_suite_utils.erl

```diff
@@ -93,27 +93,12 @@ stop_node(N, Config) ->
     cmd(["(cd ", node_shortcut(N, Config),
          " && ./bin/epoch stop)"]).
 
-mine_one_block(Node) ->
-    aecore_suite_utils:subscribe(Node, block_created),
-    StartRes = rpc:call(Node, aec_conductor, start_mining, [], 5000),
-    ct:log("aec_conductor:start_mining() (~p) -> ~p", [Node, StartRes]),
-    receive
-        {gproc_ps_event, block_created, Info} ->
-            StopRes = rpc:call(Node, aec_conductor, stop_mining, [], 5000),
-            aecore_suite_utils:unsubscribe(Node, block_created),
-            ct:log("aec_conductor:stop_mining() (~p) -> ~p", [Node, StopRes]),
-            ct:log("block created, Info=~p", [Info]),
-            ok
-    after 30000 ->
-            StopRes = rpc:call(Node, aec_conductor, stop_mining, [], 5000),
-            aecore_suite_utils:unsubscribe(Node, block_created),
-            ct:log("aec_conductor:stop_mining() (~p) -> ~p", [Node, StopRes]),
-            ct:log("timeout waiting for block event~n"
-                   "~p", [process_info(self(), messages)]),
-            error(timeout_waiting_for_block)
-    end.
-
 mine_blocks(Node, NumBlocksToMine) ->
+    mine_blocks(Node, NumBlocksToMine, 100).
+
+mine_blocks(Node, NumBlocksToMine, MiningRate) ->
+    rpc:call(Node, application, set_env, [aecore, expected_mine_rate, MiningRate],
```

Comment:
This could be tricky because it has side effects on the node doing the mining. If you find a valid reason to set back the original `expected_mine_rate` (before the mining) after the mining is finished, please comment bellow.